packages/ses/src/scope-handler.js

```diff
@@ -179,7 +180,17 @@ export const createScopeHandler = (
       );
       return undefined;
     },
-  });
+  };
+
+  // The scope handler's prototype is a proxy that throws if any trap other
+  // than get/set/has are run (like getOwnPropertyDescriptors, apply,
+  // getPrototypeOf).
+  const scopeHandler = freeze(
+    create(
+      alwaysThrowHandler,
```

Comment:
I never saw why this is an improvement. This case in particular is clearly more verbose and spread out. But ok, I won't resist.